modules/@apostrophecms/template/index.js

```diff
@@ -666,25 +666,21 @@ module.exports = {
 
         if (req.aposError) {
           // A 500-worthy error occurred already, i.e. in `pageBeforeSend`
-          return error(req.aposError, 'template');
+          return error(req.aposError);
         }
 
         try {
           content = await module.render(req, template, args);
         } catch (e) {
           // The page template threw an exception. Log where it
           // occurred for easier debugging
-          return error(e, 'template');
+          return error(e);
         }
 
         return content;
 
-        function error(e, type) {
-          let now = Date.now();
-          now = dayjs(now).format('YYYY-MM-DDTHH:mm:ssZZ');
```

Comment:
Factored this out to a method so it can also be used to log 500 errors with other causes.